### ISSUE TITLE
Allow users to manage their vouches

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-dapps-v2#4526]
Users who are signed in and part of the Giveth Verifier org will have a new option from the navbar called "My Vouches" - clicking this link will take them to the "My Attestations" page on DeVouch with the GIveth Verifier filter applied - [Example](https://devouch.xyz/profile/my-attestations?organization=0xf63f2a7159ee674aa6fce42196a8bb0605eafcf20c19e91a7eafba8d39fa0404)

![Image](https://github.com/user-attachments/assets/211b5acf-eed3-48d1-b010-cb9ed0ac02aa)

**Comment by Tosinolawale** [comment]
https://www.figma.com/design/fHpNMd8Q9iBrAr6kBj9oo5/DeVouch-Giveth-Integration?node-id=8245-7070&t=iFF5YXPy82HqOUDm-1
![user drop down](https://github.com/user-attachments/assets/f16194d0-d648-456d-adcf-b6eea463e2fc)
Will "my Vouches" be visible and active irrespective of the network user is connected to? @divine-comedian

**Comment by divine-comedian** [comment]
@Tosinolawale yes it will be visible regardless of network, however for users to edit or delete vouches they will need to connect to Optimism
